src/generators/css-filter.ts

```diff
@@ -63,7 +76,7 @@ export function cssFilterStyleheetTemplate(filterValue: string, reverseFilterVal
     });
 
     if (!frameURL) {
-        const [r, g, b] = applyColorMatrix([255, 255, 255], createFilterMatrix(config));
+        let [r, g, b] = hasChromiumIssue501582() ? [0, 0, 0] : [255, 255, 255];
```

Comment:
Use `const` since it is never changed. But ESLint should have generate a warning for that.